scripts/basic-workflow-tests.sh

```diff
@@ -292,20 +292,24 @@ test_train() {
         if [ "${BACKEND}" != "vllm" ]; then
             TRAIN_ARGS+=("--gguf-model-path" "${CACHE_HOME}/instructlab/models/${GRANITE_GGUF_MODEL}")
         fi
-
         ilab model train "${TRAIN_ARGS[@]}"
-    else
+    elif [ "$SIMPLE_TRAIN" -eq 1 ]; then
         # TODO Only cuda for now
-        TRAIN_ARGS+=("--legacy" "--device=cuda")
+        TRAIN_ARGS+=("--pipeline=simple" "--device=cuda")
         if [ "$LEGACYTRAIN" -eq 0 ]; then
             TRAIN_ARGS+=("--4-bit-quant")
         fi
         if [ "${BACKEND}" != "vllm" ]; then
             TRAIN_ARGS+=("--gguf-model-path" "${CACHE_HOME}/instructlab/models/${GRANITE_GGUF_MODEL}")
         fi
+    elif [ "$FULL_TRAIN" -eq 1 ]; then
+        # test training on a CPU not the GPU
+        TRAIN_ARGS=("--pipeline=full" "--model-path=${GRANITE_SAFETENSOR_REPO}" "--data-path=${data}" "--effective-batch-size=4" --device=cpu)
```

Comment:
Why not use the GPU here?

Reply:
full training as implemented here is intended for CPU and MPS, I think expanding the support claims to CUDA + HPU + AMD would be out of scope for this first impl.

Reply:
Wouldn't `simple` be CPU and `full` be using the GPUs, simply w/o the training lib hardware acceleration?